spec/client_spec.rb

```diff
@@ -0,0 +1,18 @@
+#grequire 'spec_helper'
+
+describe Spaceship::Client do
+	class TestClient < Spaceship::Client
+		def self.hostname
+      		"https://www.howsmyssl.com"
+    	end
+	end
+	let(:client) { TestClient.new }
+	before { WebMock.allow_net_connect! }
+	after { WebMock.disable_net_connect! }
+
+	it 'doesn\'t use weak ciphers' do
+		json=client.send('request', :get, 'a/check').body
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Surrounding space missing for operator '='.<br>Tab detected.